Source/Private/VoiceInterfaceDrift.h

```diff
@@ -32,9 +32,9 @@ class ONLINESUBSYSTEMDRIFT_API FOnlineVoiceDrift : public IOnlineVoice
 	/** State of all possible remote talkers */
 	TArray<FRemoteTalker> RemoteTalkers;
 	/** Remote players locally muted explicitly */
-	TArray<FUniqueNetIdDrift> SystemMuteList;
+	TArray<DriftID> SystemMuteList;
```

Comment:
I suppose it will work just keying these off of the ID. Alternatively there's `FUniqueNetIdWrapper` used by the base voice implementation.

Reply:
Using the wrapper it might be possible to utilize more shared code that was not available when this implementation was originally made.